cube/cube.cpp

```diff
@@ -36,7 +36,10 @@
 #include <QResizeEvent>
 
 #include <QtMath>
-#include <qpa/qplatformnativeinterface.h>
+//#include <qpa/qplatformnativeinterface.h>
+// TODO - I need to fix this below line so that you can build using the above line
+#include <QtGui/5.6.0/QtGui/qpa/qplatformnativeinterface.h>
```

Comment:
to get the xcb handle: https://github.com/heeen/qtvulkan/blob/master/cube/qvulkanview.cpp#L1182